File: java/FirebaseAllInOne/app/src/main/java/com/agroinnova/firebaseallinone/MainActivity.java
package com.agroinnova.firebaseallinone;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.agroinnova.firebaseallinone.auth.Auth;
import com.agroinnova.firebaseallinone.realtimedb.RealTimeDatabase;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);




        Button real =  findViewById(R.id.realtimeDB);
        real.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, RealTimeDatabase.class));
            }
        });

        Button auth =  findViewById(R.id.auth);
        auth.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, Auth.class));
            }
        });
    }
}

File: KotlinForAndroid/android/AnimalBios/app/src/main/java/randombgappkotlinandroid/gohool/com/animalbios/MainActivity.kt
package randombgappkotlinandroid.gohool.com.animalbios

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.View
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity(), View.OnClickListener {
    override fun onClick(v: View?) {

        when(v!!.id) {

            cheetahId.id -> {

//                var cheetahIntent = Intent(this, DetailsActivity::class.java)
//                cheetahIntent.putExtra("animal", "cheetah")
//
//                startActivity(cheetahIntent)
                Log.d("String", "Hello")

                Toast.makeText(this, "Cheetah", Toast.LENGTH_LONG).show()
            }
            lionId.id -> {
                Toast.makeText(this, "Lion", Toast.LENGTH_LONG).show()
                Log.d("String", "Hello")
//                var lionIntent = Intent(this, DetailsActivity::class.java)
//                lionIntent.putExtra("animal", "lion")
//                startActivity(lionIntent)

            }

        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val cheetah = cheetahId
        val lion = lionId


        //register our views to receive onClickListener events
        cheetah.setOnClickListener(this)
        lion.setOnClickListener(this)


    }
}
